devdox/app/schemas/git_label.py

```
@@ -92,3 +85,12 @@ def __init__(
         payload: GitLabelBase = Body(...),
     ):
         self.payload = payload
+
+
+class GitLabelDBCreateDTO(BaseModel):
+    label:str = Field(..., description="Unique identifier for the Git label")
```

Flake8 (7.2.0)

[error] missing whitespace after ':'

(E231)